### PR TITLE
Add series file compaction warning to enterprise 1.11.3 release notes

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#406

- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1/about-the-project/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,26 @@ InfluxDB Enterprise builds are available. For more information, see
 [FIPS-compliant InfluxDB Enterprise builds](/enterprise_influxdb/v1/introduction/installation/fips-compliant/).
 {{% /note %}}
 
+{{% warn %}}
+#### Series file compaction on start-up
+
+With InfluxDB Enterprise v1.11.3+, on start-up, InfluxDB run the
+`influxd_inspect -compact-series-file` command to [compacts series files](/enterprise_influxdb/v1/tools/influx_inspect/#--compact-series-file-) before data nodes are started.
```
Comment (davidby-influx) [inline content/enterprise_influxdb/v1/about-the-project/release-notes.md:28]: "compact" perhaps, not "compacts"

**Review by davidby-influx — CHANGES_REQUESTED** [review]
One possible change.

File: content/enterprise_influxdb/v1/about-the-project/release-notes.md (hunk 1 above)
Comment (sanderson) [inline content/enterprise_influxdb/v1/about-the-project/release-notes.md:28]:
```suggestion
`influxd_inspect -compact-series-file` command to [compact series files](/enterprise_influxdb/v1/tools/influx_inspect/#--compact-series-file-) before data nodes are started.
```

**Review by davidby-influx — APPROVED** [review]
LGTM!

**Pull request merged**